aea/aea.py

```diff
@@ -217,9 +213,6 @@ def update(self) -> None:
 
         :return None
         """
-        # TODO: task should be submitted by the behaviours and handlers
-        for task in self.filter.get_active_tasks():
-            task.execute()
```

Comment:
this implies removing the task registry.

Reply:
and so the dynamic loading and insantiation of `*Task` classes in `skill_folder/tasks.py`.